packages/katello/rubygem-foreman_rh_cloud/rubygem-foreman_rh_cloud.spec

```diff
@@ -5,10 +5,10 @@
 
 %global gem_name foreman_rh_cloud
 %global plugin_name rh_cloud
-%global foreman_min_version 2.3
+%global foreman_min_version 2.5
 
 Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 3.0.21.1
+Version: 4.0.21.1
```

Comment:
The change in code is very minimal, but from now on `v4` will be developed mainly and relevant cherry-picks will be added to `v3`, cc @shimshtein